test/e2e/mobileRenderIntegrity.test.ts

```diff
@@ -0,0 +1,57 @@
+import 'dotenv/config.js'
+import * as mwoffliner from '../../src/mwoffliner.lib.js'
+import rimraf from 'rimraf'
+import { execa } from 'execa'
+import { jest } from '@jest/globals'
+import { zimcheckAvailable, zimdumpAvailable, zimcheck } from '../util.js'
+
+jest.setTimeout(200000)
+
+let zimcheckIsAvailable
+let zimdumpIsAvailable
+
+beforeAll(async () => {
+  zimcheckIsAvailable = await zimcheckAvailable()
+  zimdumpIsAvailable = await zimdumpAvailable()
+})
+
+async function getOutFiles(testId: string, articleList: string, mwUrl: string): Promise<any> {
+  const parameters = {
+    mwUrl,
+    adminEmail: 'mail@mail.com',
+    outputDirectory: testId,
+    redis: process.env.REDIS,
+    articleList,
+    forceRender: 'WikimediaMobile',
+  }
+
+  await execa('redis-cli flushall', { shell: true })
+  const outFiles = await mwoffliner.execute(parameters)
+
+  return outFiles
+}
+
+const commonTreatmentTest = async (articleList: string, mwUrl: string) => {
+  if (!zimcheckIsAvailable || !zimdumpIsAvailable) {
```

Comment:
Please make a test once if zimcheck and zimdump are available and make the test dies otherwise. We should not test further if these tools are available.

Reply:
Done.